File: javascript/b1015.js
var input = require("fs").readFileSync("stdin", "utf8");

var values = input.split(/[\s\n]+/)

var x1 = parseFloat(values.shift());
var y1 = parseFloat(values.shift());
var x2 = parseFloat(values.shift());
var y2 = parseFloat(values.shift());

var sub1 = x2 - x1;
var sub2 = y2 - y1;

var pow1 = Math.pow(sub1, 2);
var pow2 = Math.pow(sub2, 2);
var distance = Math.sqrt(pow1 + pow2);

console.log(distance.toFixed(4));

File: python/b1015.py
import math;

x1, y1 = map(float, input().split(" "));
x2, y2 = map(float, input().split(" "));
formula = math.pow((x2 - x1), 2) + math.pow((y2 - y1), 2);
distance = math.sqrt(formula);

print(f"{distance:.4f}");
